### DISCUSSION TITLE
2022/05/21/%E9%92%9B%E5%90%88%E9%87%91%E4%BC%9E%E7%9A%84%E5%9E%83%E5%9C%BE%E5%9C%BA/

### DISCUSSION THREAD

**Discussion by giscus** [Pinghigh/pinghigh.github.io#1] [Announcements]
# 2022/05/21/%E9%92%9B%E5%90%88%E9%87%91%E4%BC%9E%E7%9A%84%E5%9E%83%E5%9C%BE%E5%9C%BA/

ArchLinux添加证书1234#复制根证书到信任目录下sudo cp /usr/local/share/ca-certificates/*.crt /etc/ca-certificates/trust-source/anchors#更新信任列表sudo trust extract-compat

https://pinghigh.github.io/2022/05/21/%E9%92%9B%E5%90%88%E9%87%91%E4%BC%9E%E7%9A%84%E5%9E%83%E5%9C%BE%E5%9C%BA/

**Comment by Pinghigh** [comment]
大家好

**Comment by ghost** [comment]
可不可以出一个 $Gal$ 推荐

**Reply by Pinghigh** [reply]
能玩您的谁还玩 $\textbf {Gal}$ 啊 /se
